hack/make/ubuntu

```diff
@@ -150,6 +160,7 @@ EOF
 			--deb-recommends ca-certificates \
 			--deb-recommends git \
 			--deb-recommends xz-utils \
+			--deb-recommends apparmor \
```

Comment:
That's asking for a more disruptive change for Debian users who don't have
apparmor by default.

Where is this Recommends already?  It should be only on Ubuntu in the other
scripts.